core/modulesource.go

```diff
@@ -447,10 +447,16 @@ func (src *GitModuleSource) Symbolic() string {
 }
 
 func (src *GitModuleSource) HTMLURL() string {
-	u := "https://" + src.Root + "/tree/" + src.Commit
+	u := src.CloneURL
+	if strings.HasPrefix(src.CloneURL, "https://github.com") || strings.HasPrefix(src.CloneURL, "https://gitlab.com") {
+		u = u + "/tree/" + src.Commit
+	} else {
+		u = u + "/src/" + src.Commit
+	}
```

Comment:
Bleh, the fact we need to do this is annoying. Sounds good to me though, I'm sure we'll get bug reports from users, and we can fix these one-by-one as they come in.

nit: we should use the parsed urls and look at the host FYI, I think this would break with a `http://` scheme?

Reply:
You're totally right, this has been extended ✅